gdx/src/com/badlogic/gdx/graphics/g3d/utils/Debug3dRenderer.java

```diff
@@ -0,0 +1,119 @@
+/*******************************************************************************
+ * Copyright 2011 See AUTHORS file.
+ * 
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ * 
+ *   http://www.apache.org/licenses/LICENSE-2.0
+ * 
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ ******************************************************************************/
+
+package com.badlogic.gdx.graphics.g3d.utils;
+
+import com.badlogic.gdx.graphics.GL20;
+import com.badlogic.gdx.graphics.Mesh;
+import com.badlogic.gdx.graphics.VertexAttribute;
+import com.badlogic.gdx.graphics.VertexAttributes;
+import com.badlogic.gdx.graphics.VertexAttributes.Usage;
+import com.badlogic.gdx.graphics.g3d.Material;
+import com.badlogic.gdx.graphics.g3d.Renderable;
+import com.badlogic.gdx.graphics.g3d.RenderableProvider;
+import com.badlogic.gdx.utils.Array;
+import com.badlogic.gdx.utils.Disposable;
+import com.badlogic.gdx.utils.GdxRuntimeException;
+import com.badlogic.gdx.utils.Pool;
+
+/** Debug3dRenderer allows to see box, camera in 3d space.
+ * <p>
+ * How to use it :
+ * </p>
+ * 
+ * <pre>
+ * // Create debugger
+ * Debug3dRenderer debugger = new Debug3dRenderer();
+ * // Each Frame
+ * MeshPartBuilder builder = debugger.begin();
+ * FrustumShapeBuilder.build(builder, camera);
+ * BoxShapeBuilder.build(builder, box);
+ * debugger.end()
+ * // Render
+ * modelBatch.render(debugger);
+ * // After using it
+ * debugger.dispose();
+ * </pre>
+ * 
+ * @author realitix */
+public class Debug3dRenderer implements Disposable, RenderableProvider {
+
+	/** Final renderable */
+	private final Renderable renderable = new Renderable();
```

Comment:
Javadocs should have some added value. `final renderable` doesn't say anything the code itself doesn't say. Btw, especially for private fields you don't necessarily have to add javadocs if there's nothing that needs clarification.